<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemWidgetShow</class>
 <widget class="QWidget" name="ItemWidgetShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>157</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>12</number>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="title">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Episode Name</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="episodes">
          <property name="text">
           <string>Episode Count</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="iconPoster">
          <property name="minimumSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Poster</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../data/MediaElch.qrc">:/mediaStatus/poster/green</pixmap>
          </property>
          <property name="iconGreen" stdset="0">
           <pixmap resource="../../data/MediaElch.qrc">:/mediaStatus/poster/green</pixmap>
          </property>
          <property name="iconRed" stdset="0">
           <pixmap resource="../../data/MediaElch.qrc">:/mediaStatus/poster/red</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="iconFanart">
          <property name="minimumSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../data/MediaElch.qrc">:/mediaStatus/fanart/green</pixmap>
          </property>
          <property name="iconGreen" stdset="0">
           <pixmap resource="../../data/MediaElch.qrc">:/mediaStatus/fanart/green</pixmap>
          </property>
          <property name="iconRed" stdset="0">
           <pixmap resource="../../data/MediaElch.qrc">:/mediaStatus/fanart/red</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="iconExtraFanarts">
          <property name="minimumSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../data/MediaElch.qrc">:/mediaStatus/extraFanarts/green</pixmap>
          </property>
          <property name="iconGreen" stdset="0">
           <pixmap resource="../../data/MediaElch.qrc">:/mediaStatus/extraFanarts/green</pixmap>
          </property>
          <property name="iconRed" stdset="0">
           <pixmap resource="../../data/MediaElch.qrc">:/mediaStatus/extraFanarts/red</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="iconThumb">
          <property name="minimumSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../data/MediaElch.qrc">:/mediaStatus/thumb/green</pixmap>
          </property>
          <property name="iconGreen" stdset="0">
           <pixmap resource="../../data/MediaElch.qrc">:/mediaStatus/thumb/green</pixmap>
          </property>
          <property name="iconRed" stdset="0">
           <pixmap resource="../../data/MediaElch.qrc">:/mediaStatus/thumb/red</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="iconLogo">
          <property name="minimumSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../data/MediaElch.qrc">:/mediaStatus/logo/green</pixmap>
          </property>
          <property name="iconGreen" stdset="0">
           <pixmap resource="../../data/MediaElch.qrc">:/mediaStatus/logo/green</pixmap>
          </property>
          <property name="iconRed" stdset="0">
           <pixmap resource="../../data/MediaElch.qrc">:/mediaStatus/logo/red</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="iconClearArt">
          <property name="minimumSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../data/MediaElch.qrc">:/mediaStatus/clearart/green</pixmap>
          </property>
          <property name="iconGreen" stdset="0">
           <pixmap resource="../../data/MediaElch.qrc">:/mediaStatus/clearart/green</pixmap>
          </property>
          <property name="iconRed" stdset="0">
           <pixmap resource="../../data/MediaElch.qrc">:/mediaStatus/clearart/red</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="iconCharacterArt">
          <property name="minimumSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../data/MediaElch.qrc">:/mediaStatus/actors/green</pixmap>
          </property>
          <property name="iconGreen" stdset="0">
           <pixmap resource="../../data/MediaElch.qrc">:/mediaStatus/actors/green</pixmap>
          </property>
          <property name="iconRed" stdset="0">
           <pixmap resource="../../data/MediaElch.qrc">:/mediaStatus/actors/red</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="iconBanner">
          <property name="minimumSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../data/MediaElch.qrc">:/mediaStatus/banner/green</pixmap>
          </property>
          <property name="iconGreen" stdset="0">
           <pixmap resource="../../data/MediaElch.qrc">:/mediaStatus/banner/green</pixmap>
          </property>
          <property name="iconRed" stdset="0">
           <pixmap resource="../../data/MediaElch.qrc">:/mediaStatus/banner/red</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="iconMissing">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../data/MediaElch.qrc">:/img/missing.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="iconNew">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="toolTip">
           <string>new</string>
          </property>
          <property name="pixmap">
           <pixmap resource="../../data/MediaElch.qrc">:/img/star_blue.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="iconSync">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Synchronization needed</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../data/MediaElch.qrc">:/img/reload_orange.png</pixmap>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../data/MediaElch.qrc"/>
 </resources>
 <connections/>
</ui>
